src/quantum/index-64bit.js

```diff
@@ -17,20 +15,6 @@ export default class QuantumIndex64 extends React.Component {
   constructor(props) {
     super(props);
     document.body.classList.add('multipage');
-    this.fetchNotes();
```

Comment:
When doing calls on async functions it is important to handle it properly (try/catch block).
We were getting warnings.
Since we don't use this code I removed it.